GGRC-45 UX changes in people list

AlexNo opened the pull request:
Comments from UX Designer:
We already show labels in People section (Verifier, Creator etc). No need for additional icons as we do not show them for other labels /categories. 
Comments section: it's quite unclear what icons mean (a letter means a role). Instead, I would suggest using Role after a person name 
![ggrc-45_comment_icons](https://cloud.githubusercontent.com/assets/4737102/20347915/4389498c-ac14-11e6-9803-b065bc7bbaf5.png)

Delete icon:
Currently, this icon appears with a delay and at the left - what is not common behavior. I'd suggest showing "trash" icon on mouse hover at the right. On hover - in order not to "promote" delete action in People section.
![ggrc-45_delete](https://cloud.githubusercontent.com/assets/4737102/20347955/67c27e2c-ac14-11e6-8a2a-9240fcf0c2d5.png)

Some more UI issues for this page:

- Control to add URL and People are different
<img width="1380" alt="ggrc_asmt_ui_1" src="https://cloud.githubusercontent.com/assets/4737102/20348017/a81e8e16-ac14-11e6-94f6-0e22e1106f52.png">
- different paddings after a category label and  icon
- icons have different colors: e.g. for  and  icons
- different label format
<img width="1416" alt="ggrc_asmt_ui_2" src="https://cloud.githubusercontent.com/assets/4737102/20348034/b9fa995e-ac14-11e6-8420-602e273fe2fb.png">
- separator between section disappears when one section is open

plamut commented:
@AlexNo What issues are fixed with this PR, can you please c/p the ticket description here? Thanks.
(I don't have access to Jira)

On a general note, please add tests and docstrings - at least for the Mustache helper that is potentially going  to be used at quite a few places.

AlexNo commented:
@plamut I've updated description.

plamut commented:
Thank you, @AlexNo, I can (and will) now review this PR.

AlexNo commented:
@plamut 
point 
> Control to add URL and People are different
not implemented in scope of this PR and we have created separate issue for this

plamut commented:
@AlexNo Thanks for the update, I just discovered and noted this one - I will ignore it in the review. :)

AlexNo commented:
@plamut I've fixed test cases, but don't have time for check your notices about people-list right now. I'll check it a bit letter.

plamut commented:
@AlexNo Looks good, thanks. As for the flickering issue, I suggest to show to someone and ask whether it should be addressed in this PR or not. If it is deemed minor enough to not block this PR, we can merge it and address that issue separately.

Added the `question` label to wait for the decision.

plamut commented:
@AlexNo What's the status of  this, any updates on the issue found? Is it a blocker or not?

AlexNo commented:
@plamut I've already updated PR. I've fixed a ghost copy of the the trash icon and corrected the shadow on the other side due to changing the location of icons. And fixed the issue when we cannot the hover on trash icon when have the short name in the list.

plamut commented:
@AlexNo OK, didn't know about that, sorry. Just next time please make sure to properly update the PR labels and/or post a reply, in order to not unnecessarily block the review process, thanks.
I will review the fix now.

AlexNo commented:
@plamut I've removed changes related movement trash icon to right side from scope of this PR, because we have separate issue (GGRC-470) where we have to unify all controls in the application and we avoid double work for it.

plamut commented on src/ggrc/assets/javascripts/controllers/tree_view_controller.js:
```diff
@@ -1221,6 +1221,10 @@ CMS.Controllers.TreeLoader('CMS.Controllers.TreeView', {
+    // not to react to Relationship from people list
+    if (instance.attr('attrs.AssigneeType')) {
+      return false;
+    }
```
This does not help, unfortunately. I tried to unmap a person from an Assessment, and the info pane still collapsed despite the fix here, because there is yet [another place](https://github.com/google/ggrc-core/blob/develop/src/ggrc/assets/javascripts/controllers/tree_view_controller.js#L1036-L1039) in the tree view controller that does this.

Please remove this `if` block, there is [another PR](https://github.com/google/ggrc-core/pull/4740) that tries to address that (comments and ideas highly welcome there!).

AlexNo replied:
@plamut Ok, I agree. I've removed these lines in favor your PR.
